src/Widgets/FormRatingWidget/IconEditorTab/IconSearch.tsx

```diff
@@ -0,0 +1,63 @@
+import * as React from "react";
+
+interface IconSearchProps {
+  setSearchValue: React.Dispatch<React.SetStateAction<string>>;
+  searchValue: string;
+}
+interface ClearSearchButtonProps {
+  setSearchValue: React.Dispatch<React.SetStateAction<string>>;
+  searchValue: string;
+}
+
+export const IconSearch: React.FC<IconSearchProps> = ({
+  setSearchValue,
+  searchValue,
+}) => {
+  return (
+    <div id="search">
+      <label htmlFor="search-input">
+        <i className="fa fa-search" aria-hidden="true" />
+        <span className="sr-only">Search icons</span>
+      </label>
+      <input
+        id="search-input"
+        className="form-control"
+        placeholder="Search icons"
+        autoComplete="off"
+        spellCheck="false"
+        autoCorrect="off"
+        tabIndex={1}
+        value={searchValue}
+        onChange={(event) => {
+          event.preventDefault();
+          event.stopPropagation();
+
+          setSearchValue(event.target.value);
+        }}
+      />
+      <ClearSearchButton
+        setSearchValue={setSearchValue}
+        searchValue={searchValue}
+      />
+    </div>
+  );
+};
+
+const ClearSearchButton: React.FC<ClearSearchButtonProps> = ({
+  setSearchValue,
+  searchValue,
+}) => {
+  if (!searchValue.length) return null;
+
+  return (
+    <a
```

Comment:
Does it need to be an `a` element?

Reply:
It could be changed, the thing is, that the complete IconEditorTab is just a copy of the original IconEditorTab in the Example App. The only difference is that here we have types